sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -370,28 +370,35 @@ class HiveDDLSuite
       spark.range(10).write.saveAsTable(tabName)
       val viewName = "view1"
       withView(viewName) {
+        def checkProperties(
+            properties: Map[String, String],
+            expected: Map[String, String]): Boolean = {
+          properties.filterNot { case (key, value) =>
+            Seq("transient_lastDdlTime", CatalogTable.VIEW_DEFAULT_DATABASE).contains(key) ||
+              key.startsWith(CatalogTable.VIEW_QUERY_OUTPUT_PREFIX)
```

Comment:
shall we just filter out table properties that start with `CatalogTable.VIEW_QUERY_OUTPUT_PREFIX`?